javascript/modules/client/src/internal/utils/client.ts

```diff
@@ -16,6 +23,55 @@ import {
   TransferType,
 } from "../interfaces/client";
 
+export function unwrapCreateDaoParams(
```

Comment:
Wasn't this renamed in a different way?  Wrap/Unwrap was gone

Reply:
Nope, this was renamed only on the `internal/encoding` and `internal/encoding` files. We can use the same strategy to rename them if you want and leave them in the `utils` folder or put them in a new one. 

Let me know what do you think

Reply:
OK, then let's not grow the scope of this PR and keep things as they are

Reply:
https://aragonassociation.atlassian.net/browse/APP-1138